utils/find-wasm.js

```diff
@@ -0,0 +1,42 @@
+const util = require('util');
+const glob = util.promisify(require('glob'));
+const fs = require('fs');
+
+
+
+async function findWasm(target) {
+    let res = await glob(`**/${target}/*.wasm`, { 
+        ignore: '**/node_modules/**'
+    });
+    return res;
+}
+
+async function findWasmFile(wasmFile, target) {
+    try {
+        return findWasmFileSafe(wasmFile, target);
+    } catch (err) {
+        console.error(err.message);
+        process.exit(1);
+    }
+}
+
+async function findWasmFileSafe(wasmFile, target) {
+    if (fs.existsSync(wasmFile)) {
+        return wasmFile;
+    }
+    if (wasmFile === './out/main.wasm') {
+        let res = await findWasm(target);
+        if (res.length == 0) {
+            throw new Error(`Wasm file ${wasmFile} does not exist and none could be found for target ${target}.`);
+        }
+        if (res.length > 1) {
+            throw new Error(`Multiple Wasm files found:\n\t• ${res.join('\n\t• ')}\n\nPlease pass one with --wasmFile=file`);
```

Comment:
@willemneal this PR is awesome!!1 you rock.
Even wrote a test, too. 🤩 

The only thing I think we could improve is this Error. This will give us kind of a nasty stacktrace:

<img width="1489" alt="Screen Shot 2020-12-03 at 9 27 28 PM" src="https://user-images.githubusercontent.com/1042667/101125617-82349d80-35ae-11eb-9a7a-bd6b6103e031.png">

If we change this to:

```
console.warn(`Multiple Wasm files found:\n\t• ${res.join('\n\t• ')}\n\nPlease pass one with --wasmFile=file`);
process.exit(1);
```

Then we get:

<img width="1360" alt="Screen Shot 2020-12-03 at 9 29 50 PM" src="https://user-images.githubusercontent.com/1042667/101125691-b3ad6900-35ae-11eb-910b-80bc1ebea36c.png">

Whaddayathink?

Reply:
Probably should use console.error instead of console.warn, but otherwise I like your suggestion to stop using 'throw', Mike

Reply:
That's weird because I wrote the second find function that catches the error and exits 1.  I am renaming the `findWasmFileSafe` to `findWasmFileUnsafe`, since it is the one that throws errors.  Running locally with the debugger I am realizing that the error jumps above my try catch to the caller of the safe version.

Reply:
I see my issue.  It's async.  So I need to await the result to get the value, otherwise it gets back the promise which then the caller handles.